source/opt/passes.cpp

```diff
@@ -71,5 +79,88 @@ bool FreezeSpecConstantValuePass::Process(ir::Module* module) {
   return modified;
 }
 
+bool EliminateDeadConstantPass::Process(ir::Module* module) {
```

Comment:
I have that as part of the "-load *.so" CL #271. That definitely one thing needs to do.

In #271, the abstract `pass` class is extracted to a `pass.h` file, and all passes should only needs to include that header.